contents/docs/product-analytics/sql.md

```diff
@@ -0,0 +1,293 @@
+---
+title: SQL (beta)
+availability:
+    free: full
+    selfServe: full
+    enterprise: full
+---
+
+SQL insights enable you to directly access data in PostHog using SQL queries.
+
+> SQL queries in PostHog don’t require the trailing semi-colon (`;`) of traditional SQL queries.
+
+## SQL commands
+
+You create SQL queries out of a list of commands that shape what data, filters, aggregations, and ordering we want to see. 
+
+### SELECT
+
+Use `SELECT` to select data (usually columns, transformations, or aggregations) from one or more tables in the database.
+
+```sql
+SELECT *
+FROM events
+```
+
+Common values to select are `*` (representing all), `event`, `timestamp`, `properties`, and functions. Add the `DISTINCT` clause to `SELECT` commands to keep only unique rows in query results.
```

Comment:
Where can I find what columns are possible to select?
Tbh this is my biggest difficulty with HogQL (figuring out the schema)

Reply:
Added links to properties and database tables.